src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -159,6 +159,10 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
         imageId
     }
 
+    void logConfig(String type, Map<String, String> config) {
+        logConfig = LogConfig.newInstance(type: type, config: config)
```

Comment:
No need to use the `newInstance` method as there aren't any constructors defined. You can just use `new LogConfig(type: type, config: config)`.